src/features/Classes/EnrollStudent/index.jsx

```diff
@@ -47,20 +47,47 @@ const EnrollStudent = ({ isOpen, onClose, queryClassId }) => {
     try {
       setLoading(true);
       const response = await handleEnrollments(formData, queryClassId);
+      const validationEmailList = response?.data?.results;
+      const messages = await getMessages();
+      let validEmails = [];
+      let invalidEmails = [];
+      let textToast = '';
 
       /**
        * This is because the service that checks the enrollment status is a different
        * endpoint, and that endpoint always returns a status 200, so the error cannot be
        * caught with a .catch.
        */
-      if (response?.data?.results[0]?.tags === 'error') {
-        setToastMessage(decodeURIComponent(response?.data?.results[0]?.message));
+      if (messages?.data?.results[0]?.tags === 'error') {
+        setToastMessage(decodeURIComponent(messages?.data?.results[0]?.message));
         setShowToast(true);
 
         return onClose();
       }
 
-      setToastMessage('Email invite has been sent successfully');
+      validationEmailList.map(email => {
+        if (email?.invalidIdentifier) {
+          invalidEmails = [
+            ...invalidEmails,
+            email.identifier,
+          ];
+          return invalidEmails;
+        }
+        validEmails = [
+          ...validEmails,
+          email.identifier,
+        ];
+        return validEmails;
+      });
+
+      if (invalidEmails.length > 0) {
+        textToast = `The following email adresses are invalid: ${invalidEmails.join(' ')}\n`;
+      }
+      if (validEmails.length > 0) {
+        textToast += `Successfully enrolled and sent email to the following users: ${validEmails.join(' ')}`;
+      }
+
+      setToastMessage(textToast);
```

Comment:
Something that does not convince to much to me is the error and success message are displayed in the same toast I would recommend to add more vertical space between each other at least.